src/Invoice.js

```diff
@@ -0,0 +1,165 @@
+import React, { useContext, useEffect, useState } from 'react'
+import { InvoicesContext } from "./InvoicesContext";
+import { useParams, useNavigate } from "react-router-dom";
+import iconArrowLeft from "./assets/images/icon-arrow-left.svg";
+
+const Invoice = () => {
+    const data = useContext(InvoicesContext);
+    const invoices = data;
+    const { invoiceId } = useParams();
+    const [ invoice, setInvoice ] = useState(invoices.find(inv => inv.id === invoiceId));
+
+    const navigate = useNavigate();
+
+    function goBack() {
+        navigate("/");
+    }
+
+    function deleteInvoice() {
+        const index = invoices.indexOf(invoice);
+        const invArray = invoices;
+        invArray.splice(index, 1);
+        localStorage.setItem("invoiceList", JSON.stringify(invArray));
+
+        goBack();
+    }
+
+    function payInvoice() {
+        if( invoice.status === "paid") {
+            return;
+        }
+
+        const tmpInv = invoice;
+        tmpInv.status = "paid";
+        
+        setInvoice({...invoice, tmpInv});
+        const index = invoices.indexOf(invoice);
+        invoices[index].status = "paid";
+        localStorage.setItem("invoiceList", JSON.stringify(invoices));
+    }
+
+    return (
+    <>
+    <div className="all-content light-theme">
```

Comment:
ovde uvlacenje nije dobro
```suggestion
        <div className="all-content light-theme">
```